cpp/src/traversal/bfs_kernels.cuh

```diff
@@ -874,7 +887,7 @@ __global__ void topdown_expand_kernel(
         for (int iv = 0; iv < TOP_DOWN_BATCH_SIZE; ++iv) {
           IndexType v = vec_dest_v[iv];
           vec_v_visited_bmap[iv] =
-            (v != -1) ? previous_bmap[v / INT_SIZE] : (~0);  // will look visited
+            (v != ~IndexType(0)) ? previous_bmap[v / INT_SIZE] : (~int(0));  // will look visited
```

Comment:
invalid_vertex_id<IndexType>::value is much more readable than ~IndexType(0)

Reply:
Done.